packages/playground-examples/copy/ko/4-0/New JS Features/Nullish Coalescing.ts

```diff
@@ -0,0 +1,40 @@
+// # Nullish Coalescing
+//
+// `??`는 `||`의 일반적인 사용 방법을 보완하는 새로운 연산자입니다.
+// 같은 방식으로 `===`가 `==`의 사용을 더 엄격한 등호 형태로
+// 보완시킵니다.
+//
+// 방식을 이해하기 위해 어떻게 ||가 동작하는지 봅시다:
+
+const response = {
+  nullValue: null,
+  headerText: "",
+  animationDuration: 0,
+  height: 400,
+  showSplashScreen: false,
+} as const;
+
+const undefinedValue = response.undefinedValue || "some other default";
+// 결과는 이렇게 될 겁니다: 'some other default'
```

Comment:
```suggestion
// 결과는 이렇게 됩니다: 'some other default'
```
[제안]
미래형을 현재형으로 번역하면 좀 더 자연스러워집니다.